types/gapi.client/index.d.ts

```diff
@@ -50,7 +50,7 @@ declare namespace gapi {
             //  */
             // callback?: () => any;
         // tslint:disable-next-line:no-unnecessary-generics
```

Comment:
I think this comment shoule be removed now

Reply:
Done